library/src/services/form/Field/TextField/TextField.tsx

```diff
@@ -72,10 +72,16 @@ export const TextField = (props: TextFieldProps) => {
 
   passThroughProps.onChange = (event) => {
     const { target } = event;
-
+    console.log(event.target);
+    console.log({ ...event.target });
     if (type === 'datetime-local') {
       const fixedDate = fixDateTime(target.value);
-      event.target = { ...target, ...{ value: fixedDate } };
+      event.target = {
+        ...target,
+        value: fixedDate,
+        id: event.target.id,
+        name: event.target.name,
```

Comment:
you are currently obtaining this values from target, you can just remove them

Reply:
Here's a revised version of your statement:

No, I can't.

This is the only actual change in this PR. I don't fully understand the reason behind it, but without this adjustment, I encounter a Formik warning in the console, and the date pickers become non-functional.

There must be a more elegant way to address this issue, and I'm open to any suggestions.